core/deployment/src/main/java/io/quarkus/deployment/builditem/DevServicesExposeOnDockerHostBuildItem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.deployment.builditem;
+
+import io.quarkus.builder.item.BuildItem;
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * A marker build item that, if any instances are provided during the build, the containers started by DevServices
+ * will be exposed to the Docker host. This is mainly used by {@see DevServicesKafkaProcessor} where the
+ * container may need to be exposed to Dockers host and accessible from Dockers internal network.
+ * 
+ * If a {@link DevServicesSharedNetworkBuildItem} is not produced the default behaviour is to expose services only
+ * to the Docker host and this {@link BuildItem} is not required. This {@link BuildItem} should therefore
+ * only be used in conjunction with {@link DevServicesSharedNetworkBuildItem} when expose to both the
+ * Docker host and the Docker network is required.
+ */
+public final class DevServicesExposeOnDockerHostBuildItem extends MultiBuildItem {
```

Comment:
This looks wrong, it should be a `SimpleBuildItem`

Reply:
It needs to be a `MultiBuildItem` since it can theoretically be generated by multiple extensions. In any case though, this is going to go away